andrew_eye_stream_data.py

```diff
@@ -0,0 +1,50 @@
+from talon import app, ui
+from talon_plugins.eye_mouse import tracker
+from talon.track.geom import EyeFrame
+
+main = ui.main_screen()
+
+# Lightweight simple recording for eye tracking.
+class EyeDataStream:
+    def __init__(self):        
+      self.x_coords = []
+      self.y_coords = []
+
+    def on_gaze(self, b):
+      l, r = EyeFrame(b, 'Left'), EyeFrame(b, 'Right')
+      p = (l.gaze + r.gaze) / 2
```

Comment:
Does this imply that the left and right eyes might be measured as having different fixation points?

Reply:
Yes I believe that is correct